src/kili/services/asset_import/base.py

```diff
@@ -207,9 +209,40 @@ def build_url_from_parts(*parts) -> str:
         """Builds an url from the parts."""
         return "/".join(parts)
 
+    @staticmethod
+    def are_native_videos(assets) -> bool:
+        """Determine if assets should be imported asynchronously and cut into frames."""
+        should_use_native_video_array = []
+        for asset in assets:
+            json_metadata = asset.get("json_metadata", {})
```

Comment:
You are right. It seems the default value of {} was never triggered, because we fill in the json_metadata field earlier if it is empty anyways.
Am changing this for consistency though.